openworld/sdk/generator/client/visitors/operations.py

```diff
@@ -11,21 +11,64 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import collections
+import re
 from pathlib import Path
 
+from datamodel_code_generator.model import DataModel
 from datamodel_code_generator.model.pydantic import CustomRootType
-from fastapi_code_generator.parser import OpenAPIParser, Operation
+from fastapi_code_generator.parser import Argument, OpenAPIParser, Operation
 from fastapi_code_generator.visitor import Visitor
 
 
-def get_operations(parser: OpenAPIParser, model_path: Path) -> dict[str, object]:
-    sorted_operations: list[Operation] = []
+def parse_non_schema_models(parser: OpenAPIParser):
+    models: list[DataModel] = list(
+        {
```

Comment:
1. Why do we need to check for `...endswith("Response")` and `...endswith("Request")`?
2. We uses a lot of constant strings around the codebase, can we move these to a constants file (`#/paths/`, `Response`, `Request`, ...)? 
3. This is not simple to read, we may simplify it with an external lambda function:

```python
is_non_schema_model = lambda model: isinstance(model, DataModel)
             and model.reference.path.startswith("#/paths/")
             and not model.class_name.endswith("Response")
             and not model.class_name.endswith("Request")

models: list[DataModel] = [result for result in parser.results if is_non_schema_model(result)]
```

Reply:
1. So we can skip generated root models like `GetRegionResponse`, which is not present in the spec file, but rather generated by the generator, and it has the same path `#/paths/`.